test/integration/dialects/postgres/dao.test.js

```diff
@@ -390,14 +391,23 @@ if (dialect.match(/^postgres/)) {
           return User.sync({
             logging: function (sql) {
               if (sql.indexOf('neutral') > -1) {
-                expect(sql.indexOf("ALTER TYPE \"enum_UserEnums_mood\" ADD VALUE 'neutral' BEFORE 'happy'")).to.not.be.equal(-1);
+                var cmd = semver.gte(this.sequelize.options.databaseVersion, '9.3.0')
+                            ? "ALTER TYPE \"enum_UserEnums_mood\" ADD VALUE IF NOT EXISTS 'neutral' BEFORE 'happy'"
+                            : "ALTER TYPE \"enum_UserEnums_mood\" ADD VALUE 'neutral' BEFORE 'happy'"
+                expect(sql.indexOf(cmd)).to.not.be.equal(-1);
                 count++;
               }
               else if (sql.indexOf('ecstatic') > -1) {
-                expect(sql.indexOf("ALTER TYPE \"enum_UserEnums_mood\" ADD VALUE 'ecstatic' BEFORE 'meh'")).to.not.be.equal(-1);
+                var cmd = semver.gte(this.sequelize.options.databaseVersion, '9.3.0')
+                            ? "ALTER TYPE \"enum_UserEnums_mood\" ADD VALUE IF NOT EXISTS 'ecstatic' BEFORE 'meh'"
+                            : "ALTER TYPE \"enum_UserEnums_mood\" ADD VALUE 'ecstatic' BEFORE 'meh'"
+                expect(sql.indexOf(cmd)).to.not.be.equal(-1);
                 count++;
               }
               else if (sql.indexOf('joyful') > -1) {
+                var cmd = semver.gte(this.sequelize.options.databaseVersion, '9.3.0')
+                            ? "ALTER TYPE \"enum_UserEnums_mood\" ADD VALUE IF NOT EXISTS 'joyful' AFTER 'meh'"
+                            : "ALTER TYPE \"enum_UserEnums_mood\" ADD VALUE 'joyful' AFTER 'meh'"
                 expect(sql.indexOf("ALTER TYPE \"enum_UserEnums_mood\" ADD VALUE 'joyful' AFTER 'meh'")).to.not.be.equal(-1);
```

Comment:
Please add sql unit tests, don't ammend the integration tests that test sql (we're in the procress of removing them).